demos/dermatology.html

```diff
@@ -172,13 +177,16 @@
         var mainDiv = document.getElementById("main");
         var frame = null;
         var main = null;
+        var propTree = null;
         var testTheme = null;
         var testWidget = null;
         var currWidget = null;
+        var propEditor = null;
         var showProperties = null;
+        var showPropertyTree = null;
         var debouncedResize = null;
 
-        require(["d3", "src/layout/Surface", "src/layout/Grid", "src/other/Persist", "src/other/PropertyEditor", "test/Factory"], function (d3, Surface, Grid, Persist, PropertyEditor, testFactory) {
+        require(["d3", "src/layout/Surface", "src/layout/Grid", "src/other/Persist", "src/other/PropertyEditor", "src/tree/CirclePacking", "test/Factory"], function (d3, Surface, Grid, Persist, PropertyEditor, Dendrogram, testFactory) {

```

Comment:
The Dendrogram has major issues with overlapping text, but CirclePacking visualizes the hierarchy nicely.